ccl/sqlccl: TestBackupRestoreDropDB failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/76c3503e963b26b0d31b6235339fe6397e448e20

Parameters:
```
TAGS=
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=395918&tab=buildLog

```
W171102 06:17:55.555533 101187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I171102 06:17:55.559242 101187 server/config.go:512  [n?] 1 storage engine initialized
I171102 06:17:55.559256 101187 server/config.go:515  [n?] RocksDB cache size: 128 MiB
I171102 06:17:55.559262 101187 server/config.go:515  [n?] store 0: in-memory, size 0 B
I171102 06:17:55.560950 101187 server/node.go:347  [n?] **** cluster a80be812-316c-42f1-8322-227d2d712f87 has been created
I171102 06:17:55.560970 101187 server/server.go:907  [n?] **** add additional nodes by specifying --join=127.0.0.1:40577
I171102 06:17:55.561044 101187 vendor/google.golang.org/grpc/resolver_conn_wrapper.go:64  dialing to target with scheme: ""
I171102 06:17:55.561053 101187 vendor/google.golang.org/grpc/resolver_conn_wrapper.go:70  could not get resolver for scheme: ""
I171102 06:17:55.562324 101187 storage/store.go:1214  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I171102 06:17:55.562369 101187 server/node.go:472  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=3.2 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=3298.00 p25=3298.00 p50=3298.00 p75=3298.00 p90=3298.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I171102 06:17:55.562395 101187 server/node.go:329  [n1] node ID 1 initialized
I171102 06:17:55.562439 101187 gossip/gossip.go:331  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:40577" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:1 patch:0 unstable:3 > 
I171102 06:17:55.562558 101187 storage/stores.go:332  [n1] read 0 node addresses from persistent storage
I171102 06:17:55.562619 101187 server/node.go:617  [n1] connecting to gossip network to verify cluster ID...
I171102 06:17:55.563353 101187 server/node.go:642  [n1] node connected via gossip and verified as part of cluster "a80be812-316c-42f1-8322-227d2d712f87"
I171102 06:17:55.563383 101187 server/node.go:414  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I171102 06:17:55.563422 101187 sql/distsql_physical_planner.go:124  [n1] creating DistSQLPlanner with address {tcp 127.0.0.1:40577}
I171102 06:17:55.565826 101408 vendor/google.golang.org/grpc/resolver_conn_wrapper.go:64  dialing to target with scheme: ""
I171102 06:17:55.565849 101408 vendor/google.golang.org/grpc/resolver_conn_wrapper.go:70  could not get resolver for scheme: ""
I171102 06:17:55.566680 101187 server/server.go:1119  [n1] starting https server at 127.0.0.1:52583
I171102 06:17:55.566841 101187 server/server.go:1120  [n1] starting grpc/postgres server at 127.0.0.1:40577
I171102 06:17:55.566862 101187 server/server.go:1121  [n1] advertising CockroachDB node at 127.0.0.1:40577
W171102 06:17:55.566880 101187 sql/jobs/registry.go:167  [n1] unable to get node liveness: node not in the liveness table
I171102 06:17:55.600866 101187 sql/event_log.go:106  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I171102 06:17:55.612648 101187 sql/lease.go:348  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-11-02 06:17:55.612468917 +0000 UTC
I171102 06:17:55.614547 101164 storage/replica_command.go:2821  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I171102 06:17:55.635779 101187 sql/event_log.go:106  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I171102 06:17:55.657160 101164 storage/replica_command.go:2821  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I171102 06:17:55.662596 101164 storage/replica_command.go:2821  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I171102 06:17:55.667514 101164 storage/replica_command.go:2821  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I171102 06:17:55.674288 101164 storage/replica_command.go:2821  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I171102 06:17:55.685625 101164 storage/replica_command.go:2821  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I171102 06:17:55.687133 101187 sql/event_log.go:106  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:1.1-3 User:node}
W171102 06:17:55.714962 101136 storage/intent_resolver.go:349  [n1,s1,r7/1:/{Table/System…-Max}]: failed to push during intent resolution: failed to push "sql txn implicit" id=7d69bdcd key=/Table/SystemConfigSpan/Start rw=true pri=0.05226093 iso=SERIALIZABLE stat=PENDING epo=0 ts=1509603475.687855695,0 orig=1509603475.687855695,0 max=1509603475.687855695,0 wto=false rop=false seq=7
I171102 06:17:55.716369 101164 storage/replica_command.go:2821  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I171102 06:17:55.719497 101187 sql/event_log.go:106  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I171102 06:17:55.720811 101187 server/server.go:1179  [n1] done ensuring all necessary migrations have run
I171102 06:17:55.720837 101187 server/server.go:1182  [n1] serving sql connections
I171102 06:17:55.723194 101492 sql/event_log.go:106  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:40577} Attrs: Locality: ServerVersion:1.1-3} ClusterID:a80be812-316c-42f1-8322-227d2d712f87 StartedAt:1509603475563368247 LastUp:1509603475563368247}
I171102 06:17:55.757574 101164 storage/replica_command.go:2821  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I171102 06:17:55.768666 101164 storage/replica_command.go:2821  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I171102 06:17:55.794622 101164 storage/replica_command.go:2821  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I171102 06:17:55.798407 101164 storage/replica_command.go:2821  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I171102 06:17:55.802187 101164 storage/replica_command.go:2821  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I171102 06:17:55.814042 101164 storage/replica_command.go:2821  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I171102 06:17:55.821248 101164 storage/replica_command.go:2821  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I171102 06:17:55.830551 101164 storage/replica_command.go:2821  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I171102 06:17:55.876338 101620 sql/event_log.go:106  [client=127.0.0.1:36744,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:data Statement:CREATE DATABASE IF NOT EXISTS data User:root}
I171102 06:17:55.876826 101164 storage/replica_command.go:2821  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/50 [r17]
I171102 06:17:55.881603 101620 sql/event_log.go:106  [client=127.0.0.1:36744,user=root,n1] Event: "create_table", target: 51, info: {TableName:data.bank Statement:CREATE TABLE data.bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I171102 06:17:55.888136 101164 storage/replica_command.go:2821  [split,n1,s1,r17/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r18]
I171102 06:17:55.892851 101620 sql/event_log.go:106  [client=127.0.0.1:36744,user=root,n1] Event: "drop_database", target: 50, info: {DatabaseName:data Statement:DROP DATABASE data CASCADE User:root DroppedTablesAndViews:[bank]}
I171102 06:17:55.894856 101620 sql/lease.go:348  [client=127.0.0.1:36744,user=root,n1] publish: descID=51 (bank) version=2 mtime=2017-11-02 06:17:55.894740617 +0000 UTC
I171102 06:17:55.904159 101620 sql/event_log.go:106  [client=127.0.0.1:36744,user=root,n1] Event: "create_database", target: 52, info: {DatabaseName:data Statement:CREATE DATABASE data User:root}
I171102 06:17:55.906806 101164 storage/replica_command.go:2821  [split,n1,s1,r18/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r19]
I171102 06:17:55.910698 101620 sql/event_log.go:106  [client=127.0.0.1:36744,user=root,n1] Event: "create_table", target: 53, info: {TableName:data.bank Statement:CREATE TABLE data.bank (i INT) User:root}
I171102 06:17:55.912932 101687 util/stop/stopper.go:474  quiescing; tasks left:
1      storage.split: processing replica
1      node.Node: batch
1      [async] kv.TxnCoordSender: heartbeat loop
W171102 06:17:55.913042 101443 vendor/google.golang.org/grpc/clientconn.go:934  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:40577: operation was canceled"; Reconnecting to {127.0.0.1:40577 0  <nil>}
W171102 06:17:55.913058 101443 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: context canceled
I171102 06:17:55.913088 101687 util/stop/stopper.go:474  quiescing; tasks left:
1      storage.split: processing replica
1      node.Node: batch
W171102 06:17:55.913128 101305 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: connection error: desc = "transport: authentication handshake failed: context canceled"
W171102 06:17:55.913378 101164 storage/replica.go:2774  [n1,s1,r18/1:/{Table/51-Max}] shutdown cancellation after 0.0s of attempting command [txn: d503050e], EndTransaction [/Local/Range/Table/51/RangeDescriptor,/Min)
W171102 06:17:55.913442 101164 internal/client/txn.go:549  [split,n1,s1,r18/1:/{Table/51-Max}] failure aborting transaction: writing transaction timed out or ran on multiple coordinators; abort caused by: node unavailable; try another peer
E171102 06:17:55.913475 101164 storage/queue.go:656  [split,n1,s1,r18/1:/{Table/51-Max}] unable to split [n1,s1,r18/1:/{Table/51-Max}] at key "/Table/52": node unavailable; try another peer
W171102 06:17:55.914414 101344 storage/store.go:1426  [n1,s1,r18/1:/Table/5{1-2}] unable to gossip on capacity change: node unavailable; try another peer
	backup_test.go:2268: error executing '
				CREATE DATABASE data;
				CREATE TABLE data.bank (i int);
				INSERT INTO data.bank VALUES (1);
			': pq: relation "data.bank" does not exist
```

--- a/pkg/ccl/sqlccl/backup_test.go
+++ b/pkg/ccl/sqlccl/backup_test.go
@@ -2264,12 +2264,10 @@ func TestBackupRestoreDropDB(t *testing.T) {
 	_, dir, _, sqlDB, cleanupFn := backupRestoreTestSetup(t, singleNode, numAccounts, initNone)
 	defer cleanupFn()
 
-	sqlDB.Exec(`DROP DATABASE data CASCADE`)
-	sqlDB.Exec(`
-		CREATE DATABASE data;
-		CREATE TABLE data.bank (i int);
-		INSERT INTO data.bank VALUES (1);
-	`)
+	sqlDB.Exec(`DROP DATABASE data`)
+	sqlDB.Exec(`CREATE DATABASE data`)
+	sqlDB.Exec(`CREATE TABLE data.bank (i int)`)
+	sqlDB.Exec(`INSERT INTO data.bank VALUES (1)`)
 
 	sqlDB.Exec("BACKUP DATABASE data TO $1", dir)
 	sqlDB.Exec("CREATE DATABASE data2")

